Tests/JSONTests/JSONTests.swift

```diff
@@ -105,6 +106,14 @@ class JSONTests: XCTestCase {
         let data = try json.serialize().string
         XCTAssertEqual(data, "[\"he \\r\\n l \\t l \\n o w\\\"o\\rrld \"]")
     }
+    
+    func testPermit() throws {
+        let json = try JSON(node: ["hello": "world",
+                                   "from":"🚀"])
```

Comment:
It looks like Xcode intended this line. Could you maybe change it to the same style as the others, please? Thanks!